create_civet_image.sh

```
@@ -578,8 +603,11 @@ if [[  ${#_arg_colourmap[@]} == 2 ]]; then
     threshold_high_left=$(awk 'function abs(x) { return ( (x < 0) ? -x : x ) } { print abs($0) }' ${tmpdir}/left | sort -g | tail -1 || true)
   fi
 
-  colour_object ${_arg_left_obj} ${tmpdir}/left ${tmpdir}/left_negative.obj ${colourmap_neg} -${threshold_low_left} -${threshold_high_left} white
-  colour_object ${tmpdir}/left_negative.obj ${tmpdir}/left ${tmpdir}/left_unmasked.obj ${colourmap_pos} ${threshold_low_left} ${threshold_high_left} transparent
+  colour_object ${_arg_left_obj} ${tmpdir}/left ${tmpdir}/left_negative.obj ${colourmap_neg} 0 -${threshold_high_left} gray "${colourmap_neg_max}" 1 replace
```

Shellcheck (0.10.0)

[warning] _arg_left_obj is referenced but not assigned.

(SC2154)